### PR TITLE
Add Touch and Interactions

### PR DESCRIPTION
**Pull request by JiaUnity**
ADD: touch test
ADD: Interactions test
FIX: Some error due to new naming

### CODE REVIEW

**Review by Rene-Damm — APPROVED** [review]

File: Assets/QA/Input_Test/Scripts/Input/DualShockISX.cs [hunk 1]
```diff
@@ -19,19 +19,19 @@ void Start()
         //m_discreteButtonAction.canceled += callbackContext => OnControllerButtonPress(callbackContext.control as ButtonControl, isPS: true);
         //m_discreteButtonAction.Enable();
 
-        m_buttonAction = new InputAction(name: "DualShockButtonAction", binding: "*DualShock*/<button>");
+        m_buttonAction = new InputAction(name: "DualShockButtonAction", binding: "*DualShock*/<button>") { passThrough = true }; ;
         m_buttonAction.performed += callbackContext => OnControllerButtonPress(callbackContext.control as ButtonControl, isPS: true);
-        m_buttonAction.canceled += callbackContext => OnControllerButtonPress(callbackContext.control as ButtonControl, isPS: true);
+        //m_buttonAction.cancelled += callbackContext => OnControllerButtonPress(callbackContext.control as ButtonControl, isPS: true);
```
Comment (jechter) [inline Assets/QA/Input_Test/Scripts/Input/DualShockISX.cs]: Why did you comment this (and the ones below) out? This API has been renamed to match the preferred US spelling, it should work using "canceled".

File: Assets/QA/Input_Test/Scripts/Input/GamepadISX.cs [hunk 2]
```diff
@@ -61,7 +61,7 @@ protected virtual void OnControllerButtonPress(ButtonControl control, string dpa
                 else if (isPS) buttonName = control.aliases[1];
                 else           buttonName = control.name.Replace("button", "");
             }
-            Debug.Log(dpadName);
+            //Debug.Log(dpadName);
```
Comment (jechter) [inline Assets/QA/Input_Test/Scripts/Input/GamepadISX.cs]: Can we rather delete unneeded code then keep it around in comments?

File: Assets/QA/Input_Test/Scripts/Input/XboxISX.cs [hunk 3]
```diff
@@ -14,19 +14,19 @@ void Start()
     {
         //m_stickMaxMove = 0.25f;
 
-        m_buttonAction = new InputAction(name: "XboxButtonAction", binding: "XInputController*/<button>");
+        m_buttonAction = new InputAction(name: "XboxButtonAction", binding: "XInputController*/<button>") { passThrough = true }; ;
         m_buttonAction.performed += callbackContext => OnControllerButtonPress(callbackContext.control as ButtonControl, isXbox: true);
-        m_buttonAction.canceled += callbackContext => OnControllerButtonPress(callbackContext.control as ButtonControl, isXbox: true);
+        //m_buttonAction.cancelled += callbackContext => OnControllerButtonPress(callbackContext.control as ButtonControl, isXbox: true);
```
Comment (jechter) [inline Assets/QA/Input_Test/Scripts/Input/XboxISX.cs]: Same here

File: Assets/QA/Input_Test/Scripts/InputUIPicker.cs [hunk 4]
```diff
@@ -62,79 +62,90 @@ void Start()
...
                 break;
+            default:
+                break;                
         }
+        m_otherDropdown.value = 0;
+    }
+
+    public void SwitchToOtherTest(int value)
+    {
+        //switch (value)
```
Comment (jechter) [inline Assets/QA/Input_Test/Scripts/InputUIPicker.cs]: can we delete this commented code?

File: Assets/QA/Input_Test/Scripts/TouchDebugInfo.cs [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+                infoSection.gameObject.SetActive(true);
+            infoSection.GetComponent<Text>().text = content;
+        }       
+
+        // Disable the ones not used
+        for (int i = index + 1; i < infoPool.childCount; i++)
+            infoPool.GetChild(i)?.gameObject.SetActive(false);                
+                    
+
+        //// Add a new section
```
Comment (jechter) [inline Assets/QA/Input_Test/Scripts/TouchDebugInfo.cs]: Same?

**Review by jechter — CHANGES_REQUESTED** [review]

**Pull request closed without merging**